platform/android/MapboxGLAndroidSDKTestApp/build.gradle

```diff
@@ -10,6 +10,7 @@ android {
         versionCode 13
         versionName "6.0.0"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        //testInstrumentationRunnerArgument 'size', 'large'
```

Comment:
Leftover? :)

Reply:
actually not, I took my a while to figure out this arg :upside_down_face: so I was like, let's keep it there for future reference.

Reply:
I'm going to remove, we need command line support for this not a gradle configuration

Reply:
:ok_hand: